dockerGrader/test/p10/requests.py

```diff
@@ -0,0 +1,9 @@
+import sys
+import importlib
+sys.path = sys.path[1:]
+def get(*args, **kwargs):
```

Comment:
what if student uses request.post, or something else?  Don't we need to add everything else in the real requests to our fake requests?  Maybe by iterating over dir(requests)?